axis.go

```diff
@@ -407,14 +417,74 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 		if val >= min && val <= max && !found {
 			ticks = append(ticks, Tick{Value: val})
 		}
-		if math.Nextafter(val, val+minorDelta) == val {
-			break
-		}
 		val += minorDelta
 	}
 	return
 }
 
+func removeDuplicates(elements []float64) []float64 {
+	// Uses a map to record duplicates as we find them.
+	seen := make(map[float64]bool)
```

Comment:
Note that elements are in an order, so you can just iterate over elements and retain elements that are not the same as the previous one.

Reply:
I tried making that 
`for _, v := range elements {
		for _, w := range elements[1:] {
			if v == w {
				continue
			}
		}
		out = append(out, v)
	}
	return out
}`
but it does not work. I do realise that it takes much less memory than the method offered by me, but here the arrays are relatively small, should not be a problem, right? Anyway, I will think why the level of precision get distorted.

Reply:
You only need to check whether the previous matches the current:
```
out := make([]float64, 1, len(elements))
out[0] = elements[0]
for i, v := range elements[1:] {
		if v == elements[i] {
			continue
		}
		out = append(out, v)
}
return out
```